lib/helper/Playwright.js

```diff
@@ -1556,21 +1551,16 @@ class Playwright extends Helper {
    * ```
    */
   async clearField(locator, options = {}) {
-    let result;
-    const isNewClearMethodPresent = typeof this.page.locator().clear === 'function';
+    const els = await findFields.call(this, locator);
+    assertElementExists(els, locator, 'Field to clear');
+
+    const el = els[0];
 
-    if (isNewClearMethodPresent) {
-      const els = await findFields.call(this, locator);
-      assertElementExists(els, locator, 'Field to clear');
-      // TODO: locator change required after #3677 implementation
-      const elXpath = await getXPathForElement(els[0]);
+    highlightActiveElement.call(this, el, this.page);
 
-      await this.page.locator(elXpath).clear(options);
-      result = await this._waitForAction();
-    } else {
-      result = await this.fillField(locator, '');
-    }
-    return result;
+    await el.fill('');
```

Comment:
why we don't use `clear` in this case?
`fill('')` could produce different result